src/Whoops/Exception/FrameCollection.php

```diff
@@ -164,11 +172,22 @@ public function serialize()
      * @see Serializable::unserialize
      * @param string $serializedFrames
      */
+    #[ReturnTypeWillChange]
     public function unserialize($serializedFrames)
     {
         $this->frames = unserialize($serializedFrames);
     }
 
+    public function __serialize(): array
+    {
+        return $this->frames;
+    }
+
+    public function __unserialize(array $serializedFrames): void
+    {
+        $this->frames = $serializedFrames;
+    }
```

Comment:
Are these related to the ReturnTypeWillChange or is this an accidental addition?

Reply:
This is also needed for PHP 8.1 compatibility because `Serializable` is deprecated.

Reply:
Thanks!